lib/plugins/aws/package/compile/functions.js

```diff
@@ -629,6 +629,30 @@ class AwsCompileFunctions {
       }
     }
 
+    if (functionObject.loggingConfig) {
+      const loggingConfig = {};
+
+      if (functionObject.loggingConfig.applicationLogLevel) {
+        loggingConfig.ApplicationLogLevel = functionObject.loggingConfig.applicationLogLevel;
+      }
+
+      if (functionObject.loggingConfig.logFormat) {
+        loggingConfig.LogFormat = functionObject.loggingConfig.logFormat;
+      }
+
+      if (functionObject.loggingConfig.logGroup) {
+        loggingConfig.LogGroup = functionObject.loggingConfig.logGroup;
+      }
+
+      if (functionObject.loggingConfig.systemLogLevel) {
+        loggingConfig.SystemLogLevel = functionObject.loggingConfig.systemLogLevel;
+      }
+
+      if (Object.keys(loggingConfig).length) {
+        functionResource.Properties.LoggingConfig = loggingConfig;
+      }
+    }
+
```

Comment:
I think this could be more compact as:
```
if (functionObject.loggingConfig) {
  const loggingConfig = {};

  ({
    ApplicationLogLevel: loggingConfig.applicationLogLevel,
    LogFormat: loggingConfig.logFormat,
    LogGroup: loggingConfig.logGroup,
    SystemLogLevel: loggingConfig.systemLogLevel,
  } = functionObject.loggingConfig);

  functionResource.Properties.LoggingConfig = Object.keys(loggingConfig).length ? loggingConfig : undefined;
}
```